wrappers/c/zxing.h

```diff
@@ -0,0 +1,164 @@
+#ifndef _ZXING_C_H
+#define _ZXING_C_H
+
+/*
+ * <stdbool.h>
+ *  bool
+ *
+ * <stdint.h>
+ *  uint8_t
+ */
```

Comment:
Please remove this comment block and instead add one with copyright and license info like all other source files have. This also applies to the other 2 new files from you.

Reply:
As far as I'm concerned, the code I'm contributing is public domain, you can do with it whatever you want (hopefully good :)

Do you want me to add a copyright notice with your name?

```
/*
* Copyright 2023 Axel Waggershauser
*/
// SPDX-License-Identifier: Apache-2.0
```

Reply:
You are the author, so your name is the correct one (in my opinion ;)).

Reply:
I added the copyright comment and used 'siiky', maybe you want to replace that with your real name.